api/krusty/testdata/localize/remote/kustomization.yaml

```diff
@@ -5,5 +5,7 @@ resources:
   - https://github.com/kubernetes-sigs/kustomize//api/krusty/testdata/localize/simple?submodules=0&ref=kustomize/v4.5.7&timeout=300
   - hpa.yaml
 
-commonLabels:
-  purpose: remoteReference
+labels:
+- includeSelectors: true
```

Comment:
I think the example should be simple.
Can you remove 'includeSelector' field from every place you think unnecessary?